08.Third-party-API-wrappers.md

```diff
@@ -5,8 +5,12 @@ You can also access the Gitter API through one of the 3rd party libraries that m
 ## .NET (C#)
 
 * [Gitter#](https://github.com/Odonno/gitter-api-pcl) - provides a Gitter API interface for use in Windows (Phone) apps, .NET Framework and Xamarin platforms
-	* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
+* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
 
 ## Java
 
 * [GitterRxJavaSDK](https://github.com/Gitteroid/GitterRxJavaSDK) - provides you a way to use Gitter API in Java with [RxJava](https://github.com/ReactiveX/RxJava) syntax
+
+## Go
+
+* [gitter](https://github.com/sromku/gitter) - Gitter API in Go. 
```

Comment:
@MadLittleMods yep, agree. Friends of mine, also noted that `go-` prefix will make more sense. So, it's changed now to **go-gitter** :) [documentation](https://godoc.org/github.com/sromku/go-gitter)